plugins/CoreHome/vue/src/externalLink.ts

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+/**
+ * Takes a raw URL and returns an HTML link tag for the URL, if the URL is for a matomo.org
+ * domain then the URL will be modified to include campaign parameters
+ *
+ * @param url     URL to process
+ * @param values  Optional [campaignOverride, sourceOverride, mediumOverride]
+ * @return string
+ */
+export function externalRawLink(
+  url: string,
+  ...values: (string|null)[]
+): string {
+  const pkArgs = values as (string|null)[];
+  return window._pk_externalRawLink(url, pkArgs); // eslint-disable-line
```

Comment:
Do we need to check if the `_pk_externalRawLink` method is available on the `window` object and if not, just return the original `url`?

Reply:
I've added a check for this :+1: